### PR TITLE
[FONTVIEW] Update Polish (pl-PL) translation

### PR DESCRIPTION
**Pull request by pithwz**
[FONTVIEW] Update Polish (pl-PL) translation

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-polish for review

File: base/applications/fontview/lang/pl-PL.rc [hunk 1]
```diff
@@ -10,20 +11,20 @@ LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
...
     IDS_ERROR_NOCLASS "Nie udało się zainicjować klasy window."
-    IDS_ERROR_ISINSTALLED "This font is already installed!" 
-    IDS_ERROR_FONTCPY "Failed to copy the font file!"
-    IDS_ERROR_OPENKEY "Failed to open the fonts key!"
-    IDS_ERROR_REGISTER "Failed to register the new font!"
-    IDS_SUCCESS "Success"
-    IDS_COMPLETED "Font installation completed."
+    IDS_ERROR_ISINSTALLED "Ta czcionka jest już zainstalowana!" 
+    IDS_ERROR_FONTCPY "Nie można skopiować pliku czcionki!"
+    IDS_ERROR_OPENKEY "Nie można otworzyć pliku czcionki!"
```
Comment (Saibamen) [inline base/applications/fontview/lang/pl-PL.rc:24]:
```suggestion
    IDS_ERROR_OPENKEY "Nie można otworzyć klucza czcionek!"
```

Comment (archeYR) [inline base/applications/fontview/lang/pl-PL.rc:24]: "fonts key" is not the same thing as "font file", and by grepping the code with `IDS_ERROR_OPENKEY` identifier, we can see that this error means that the system fonts registry key could not be opened (if it wasn't already clear from english string):
```
     /* Open the fonts key */
     res = RegOpenKeyExW(HKEY_LOCAL_MACHINE,
                         L"SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts",
                        0,
                        KEY_ALL_ACCESS,
                        &hKey);
     if (res != ERROR_SUCCESS)
     {
         ErrorMsgBox(hwnd, IDS_ERROR_OPENKEY);
          return -1;
     }
```
```suggestion
    IDS_ERROR_OPENKEY "Nie można otworzyć klucza rejestru czcionek!"
```

Comment (binarymaster) [reply base/applications/fontview/lang/pl-PL.rc]: Indeed, the English wording of it is not very clear, I also made it explicit about the registry in ru-RU translation.

**Review by archeYR — APPROVED** [review]

**Pull request merged**